src/main/resources/META-INF/NOTICE

```diff
@@ -19,3 +19,11 @@ To find the details that apply to this artifact see the accompanying LICENSE fil
 A list of contributors may be found from CREDITS(-2.x) file, which is included
 in some artifacts (usually source distributions); but is always available
 from the source code management (SCM) system project uses.
+
+## FastDoubleParser
+
+jackson-core bundles a shaded copy of FastDoubleParser <https://github.com/wrandelshofer/FastDoubleParser>.
+That code is available under an MIT license <https://github.com/wrandelshofer/FastDoubleParser/blob/main/LICENSE>
+under the following copyright.
+
+Copyright © 2023 Werner Randelshofer, Switzerland. MIT License.
```

Comment:
Techincally speaking, the copyright is `<YEAR> <COPYRIGHT HOLDER>`. The current license does not mention `2023 Werner Randelshofer, Switzerland`

Reply:
his source headers have this - so I'd prefer to err on the side of caution

Hopefully, Werner will update the LICENSE text to include this copyright too and I will then adjust the FastDoubleParser-LICENSE file that I have created here to match any changes to the one in FastDoubleParser project.

Reply:
In practice, if Werner includes the proper license file, then you'll get it automatically when shading FastDoubleParser. Then it would be easier for you to maintain since you could count on FDP to ship the up-to-date license always.

That is the reason why I ask projects to ship their license texts under `META-INF/`.

Reply:
If you hard-code 2023 here, you'll have a hard time maintaining the proper copyright header 🤷

Reply:
It could have form of 2023+ (or such), does not need to be updated every year.
If adding license straight as part of shading that'd be nice of course but I don't think we have to get super clever here. The intent of license matters, not just specific form.

Reply:
I intend to update this once we can get changes made in Fastdoubleparser license file. This PR means Jackson is not blocked from making releases. We can put a more permanent solution in place when next fastdoubleparser release is done.